src/Electron/main.js

```diff
@@ -127,9 +126,9 @@ function handleSetConfig(event, config) {
  * @param {Event} event The Electron renderer event
  * @param {Object} trigger The metadata for the event code trigger
  * @param {string} trigger.comName The COM name of the serial port
- * @param {Object} trigger.eventCodes The list of possible event codes to be triggered
  * @param {string} trigger.productID The name of the product connected to the serial port
  * @param {string} trigger.vendorID The name of the vendor connected to the serial prot
+ * @param {Object} trigger.settings The list of possible event with relative event codes to be triggered
```

Comment:
since codes are now embedded under the settings.json, I had to import the entire settings object into the trigger object to access the code